docs/general/web2-to-web3.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Web2 to Web3 - The Anatomy of a dApp
+description: Overview of the anatomy of a dApp and its architecture difference from a web2 app.
+---
+
+Overview of the anatomy of a dApp and its architecture difference from a web2 app.
+
+---
+
+## Introduction
+
+_This article assumes you have experience writing front-ends for web apps (called web2 apps here) and want to gain an an understanding of the differences and commonalities when creating front-ends for web3 apps._
+
+Web3 dApps expand on the front-end architectural systems of Web2 Applications. Concepts like SinglePageApp, State and UI Management, Data Fetching and Caching serve as the foundation of any Web3 dApp. Compared to Web2 Apps, even more computation is pushed to the client/browser as computing on the blockchain costs literal money while running your own traditional backend between the blockchain and frontend increases centralization and is not used by most web3 dApps.
+
+
+Typical DApp Stack looks something like this:
+
+![](/img/doc-images/general/web2-web3.svg)
+
+## Web2 Foundation
+
+**App Framework** — Something like Next.js, create-react-app. 
+
+**UI Management**  — As in web2, React is the most popular, Vue also is used. 
+
+**State Management** —- This is intricately tied to your UI Management Library. For the biggest dApps redux is popular.
+
+## Web3 Specific
+
+Just as in web2 where there is overlap and tie in between App Framework, UI and State Management, the web3 tools may blur boundaries especially between the Web3 Helper Libraries and Wallet Connection Manager categories.
+
+The three categories below are not  industry terms. Nevertheless It can be helpful to understand them as groupings of functionality.  Lets look at these from bottom to top, where higher means a tool will require a specific lower tool.
+
+### EVM Blockchain  Toolkit
+
+Examples — [Web3.js](https://web3js.org/) or [Ethers.js](https://ethers.org/).
+
+Web3.js has significant disadvantages on browsers due to its giant size and lack of modularity. New projects use ethers.js) These make the calls from your dapp to contracts you read and write to on the blockchain via JSON-RPC calls to a blockchain node.  These tools can run both in node.js and browsers.
+
+### Web3 Helper Libraries (optional but helpful)
+
+Examples — Wagmi, useDapp, etc.
+
+These sit between **EVM** **Blockchain  Toolkit**  and **Wallet Connection Manager** , or are in some cases are used by or have the functionality of a **Wallet Connection Manager**. They help simplify the complexities interacting with smart contracts which has additional UX considerations compared to a http api call. For instance rather than just wait for call to be returned we must 1) wait for wallet to sign 2) wait for tx to be sent to node  3) wait for it to be included in a block.
+
+### Wallet Connection Manager
+
+Examples – Rainbowkit, web3modal, react-celo.
```

Comment:
Consider removing react-celo (since we probably won't recommend folks use it going forward)

Before:
> Examples – Rainbowkit, web3modal, react-celo.

After: 
> Examples – Rainbowkit and web3modal.